app/src/main/res/values/strings.xml

```diff
@@ -460,8 +460,8 @@ Upload your first media by tapping on the add button.</string>
   <string name="images_featured_explanation">Featured pictures are images from highly skilled photographers and illustrators that the Wikimedia Commons community has chosen as some of the highest quality on the site.</string>
   <string name="images_via_nearby_explanation">Images Uploaded via Nearby places are the images which are uploaded by discovering places on the map.</string>
   <string name="thanks_received_explanation">This feature allows editors to send a Thank you notification to users who make useful edits – by using a small thank link on the history page or diff page.</string>
-  <string name="copy_image_caption_description">Copy to subsequent Media</string>
-  <string name="copied_successfully">Copied Successfully</string>
+  <string name="copy_image_caption_description">Copy to subsequent media</string>
+  <string name="copied_successfully">Copied</string>
```

Comment:
Thanks for remembering this!
Tip: You could also have added it to the other pull request, simply by pushing to the same branch and requesting another pull :-)

Reply:
Thanks for the tip, will definitely remember it if such a thing happens next time :)